packages/xo-web/src/common/utils.js

```diff
@@ -650,3 +655,22 @@ export const hasLicenseRestrictions = host =>
   host.productBrand !== 'XCP-ng' &&
   versionSatisfies(host.version, '>=7.3.0') &&
   host.license_params.sku_type === 'free'
+
+// ===================================================================
+
+export const AssertConsistentHostServerTime = decorate([
```

Comment:
Let's move it to its own module to avoid have a big file with a lot of functions.